boards/arm/nrf7002dk_nrf5340/nrf5340_cpuapp_common.dts

```diff
@@ -145,6 +145,32 @@ arduino_i2c: &i2c1 {
 	pinctrl-names = "default", "sleep";
 };
 
+&spi4 {
+	compatible = "nordic,nrf-spim";
```

Comment:
There is no other choice for `spi4`, so this is redundant here.

Reply:
All board DTS files set it, to maybe it's better to leave it for consistency even if it's redundant?

Reply:
Okay, let it stay.